src/main/java/fact/io/hdureader/Reader.java

```diff
@@ -72,5 +72,12 @@
      */
     OptionalTypesMap<String, Serializable> getNextRow() throws IOException;
 
+    /**
+     * Skips the given number of rows.
+     *
+     * @param num The amount of rows to skip.
+     * @throws IOException
+     */
+    void skipToRow(int num) throws IOException;
```

Comment:
It skips 'num' rows right? So if i call 'skipToRow(4)' twice in a row im looking at row 8 right? I propose to change the name of the method to simply 'skipRows'

Reply:
It skipped to the row from the beginning and could only be called from there, I changed it now to be able to skip rows whenever